src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2464,6 +2473,7 @@ private void ShowGetStartedGuidedTour()
             //We pass the root UIElement to the GuidesManager so we can found other child UIElements
             try
             {
+                shortcutBar.AreMenuItemsEnabled  = false;
```

Comment:
Should we use events here instead of setting from DynamoView directly? `OnEnableShortcutBarItems(false)`?